extensions/markdown-language-features/src/languageFeatures/pathCompletions.ts

```diff
@@ -181,6 +181,24 @@ export class MdVsCodePathCompletionProvider implements vscode.CompletionItemProv
 			};
 		}
 
+		const angleBrackets = /^\s*\[(.*?)\]\(\s*<(.*?)>\s*$/;
```

Comment:
This regular expression should match up the cursor inside the link. Right now it tries to match the entire link (Notice how the normal link regex above doesn't require `^\s*`) at the start